js/components/question-for-class.js

```diff
@@ -32,22 +34,30 @@ export default class QuestionForClass extends Component {
     onSelectChange(question.get('key'), event.target.checked)
   }
 
+  handleStickyStateChange(isSticky) {
+    this.setState({isSticky: isSticky})
+  }
+
   render() {
     const { question } = this.props
-    const { answersVisible } = this.state
+    const { answersVisible, isSticky } = this.state
     return (
-      <div className={`question ${question.get('visible') ? '' : 'hidden'}`}>
-        <div className='question-header'>
-          <input type='checkbox' checked={question.get('selected')} onChange={this.handleCheckboxChange}/>
-          Question #{question.get('questionNumber')}
-          <a className='answers-toggle' onClick={this.toggleAnswersVisibility}>
-            {answersVisible ? 'Hide responses' : 'Show responses'}
-          </a>
+      <StickyContainer>
+        <div className={`question ${question.get('visible') ? '' : 'hidden'}`}>
+          <Sticky className="sticky-question-header" onStickyStateChange={this.handleStickyStateChange.bind(this)}>
```

Comment:
I'd rather bind handlers in the constructor, just once in the component lifetime. If we call `.bind()` each time we render the parent component, it means we will always re-render this subcomponent, even if it uses [pureRenderMixin](https://facebook.github.io/react/docs/pure-render-mixin.html) (as `.bind` returns a new function each time it's called, so properties comparison will detect a change).
Not too important here (we don't know if `Sticky` is using this optimization or not, probably not), although generally I'd stick to this pattern since all our components use this mixin.